src/components/Results/Results.tsx

```diff
@@ -1,4 +1,6 @@
-import { makeStyles, Grid, IconButton } from '@material-ui/core';
+import { makeStyles, Grid, IconButton, AppBar, Tab, Tabs } from '@material-ui/core';
+import TabPanel from '@material-ui/lab/TabPanel';
+import TabContext from '@material-ui/lab/TabContext';
```

Comment:
doesn't matter at all, but technically these can be combined into one import, saving you a line

```suggestion
import { TabContext, TabPanel } from '@material-ui/lab';
```